app/routes/events/EventDetailRoute.js

```diff
@@ -101,7 +101,7 @@ const mapStateToProps = (state, props) => {
   return {
     comments,
     actionGrant,
-    notLoading: !state.events.fetching,
+    notLoading: !state.events.fetching && !state.users.fetching,
```

Comment:
What is the point of this?

Reply:
Just thought I wanted to wait with drawing everything until after your user is fetched, didn't really do anything though, hehe (TODO remove)